core/src/test/java/feast/core/service/JobServiceTest.java

```diff
@@ -134,14 +134,14 @@ public void setupJobRepository() {
     when(this.jobRepository.findAll()).thenReturn(Arrays.asList(this.job));
   }
 
-  // TODO: setup fake job manager
+  // setup fake job manager
```

Comment:
Can we please use Javadoc style comments like this: https://stackoverflow.com/questions/20869421/is-there-a-way-to-write-a-single-line-javadoc-comment-like-in-doxygen?

Reply:
Removed offending comments.